tests/unit/amo/components/TestAddon.js

```diff
@@ -461,6 +461,26 @@ describe(__filename, () => {
     sinon.assert.callCount(fakeDispatch, 1);
   });
 
+  // The reason for this test case came from https://github.com/mozilla/addons-frontend/issues/4541.
+  it('does not dispatch a server redirect when slug is a stringified integer that starts with -', () => {
+    const clientApp = CLIENT_APP_FIREFOX;
+    const { store } = dispatchClientMetadata({ clientApp });
+    const addon = createInternalAddon({
+      ...fakeAddon,
+      slug: '-1234',
+    });
+
+    store.dispatch(_loadAddons({ addon }));
+
+    const fakeDispatch = sinon.spy(store, 'dispatch');
+    renderComponent({
+      params: { slug: `${addon.slug}` }, store,
```

Comment:
so I guess, we could write: `{ slug: addon.slug },` no?

Reply:
🤦‍♂️ yes...